rskj-core/src/test/java/co/rsk/peg/federation/FederationStorageProviderImplTests.java

```diff
@@ -1186,6 +1186,31 @@ void saveNextFederationCreationBlockHeight_shouldSaveToStorage_whenHeightIsNotNu
 
     }
 
+    @Test
+    void save_saveLastRetiredFederationP2SHScript_afterRSKIP186_getsValueFromStorage() {
```

Comment:
remove the `save_` prefix

Reply:
`save` is the method name we are testing. We might leave the `save_` prefix and remove it from what comes after, thus making clear what method we are testing. E.g.:

`save_LastRetiredFederationP2SHScript_afterRSKIP186_getsValueFromStorage()
`
WDYT guys? @julia-zack @jeremy-then

Reply:
if he removes the `save_` prefix, the test name would be `saveLastRetiredFederationP2SHScript_afterRSKIP186_getsValueFromStorage`.

Reply:
Updated.